src/routes/safe/store/actions/fetchSafe.js

```diff
@@ -59,16 +59,21 @@ export const checkAndUpdateSafe = (safeAddress: string) => async (dispatch: Redu
   localSafe.threshold = threshold.toNumber()
 
   dispatch(updateSafeThreshold({ safeAddress, threshold: threshold.toNumber() }))
+
   // If the remote owners does not contain a local address, we remove that local owner
   localOwners.forEach((localAddress) => {
-    if (!remoteOwners.includes(localAddress)) {
+    const remoteLowerCased = remoteOwners.map((owner) => owner.toLowerCase && owner.toLowerCase())
```

Comment:
we have an util function called `sameAddress`, I think this can be rewritten as:
```
if (remoteOwners.findIndex(remoteAddress => sameAddress(remoteAddress, localAddress)) {
```

Reply:
done, thanks!